### PR TITLE
Dame Dane Guy Outfit

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
Adds Kazuma Kiryu's outfit from the Yakuza series. Includes shoes, outfit and coat. Can be found in the autodrobe.

**Media**

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Aleksh
- add: Added Dame Dane clothing set. EEYIKES!

### CODE REVIEW

**Comment by brainfood1183** [comment]
you have two entities with the same id. Clothinguniformjumpsuitsdamedane is given to both the jumpsuit and jumpskirt. you should change the id of the jumpskirt to clothinguniformjumpskirtdamedane. Then you would likely have to add that id to the clothingvend.

File: Resources/Textures/Clothing/OuterClothing/Coats/damedanecoat.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by Alekshhh, original design by SEGA",
```
Comment (AJCM-git) [inline Resources/Textures/Clothing/OuterClothing/Coats/damedanecoat.rsi/meta.json:4]:
```suggestion
  "copyright": "Made by Alekshhh (Github) for SS14, inspired by Kazuma Kiryu outfit",
```

File: Resources/Textures/Clothing/Shoes/Misc/damedaneshoes.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by Alekshhh, original design by SEGA",
```
Comment (AJCM-git) [inline Resources/Textures/Clothing/Shoes/Misc/damedaneshoes.rsi/meta.json:4]:
```suggestion
  "copyright": "Made by Alekshhh (Github) for SS14, inspired by Kazuma Kiryu outfit",
```

File: Resources/Textures/Clothing/Uniforms/Jumpsuit/damedaneoutfit.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by Alekshhh, original design by SEGA",
```
Comment (AJCM-git) [inline Resources/Textures/Clothing/Uniforms/Jumpsuit/damedaneoutfit.rsi/meta.json:4]:
```suggestion
  "copyright": "Made by Alekshhh (Github) for SS14, inspired by Kazuma Kiryu outfit",
```

**Review by AJCM-git — CHANGES_REQUESTED** [review]

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**